packages/clerk-js/src/ui/elements/OrganizationAvatar.tsx

```diff
@@ -1,20 +1,18 @@
 import type { OrganizationResource } from '@clerk/types';
 
-import { useOptions } from '../contexts';
 import { Avatar } from '../elements';
 import type { PropsOfComponent } from '../styledSystem';
 
 type OrganizationAvatarProps = PropsOfComponent<typeof Avatar> &
-  Partial<Pick<OrganizationResource, 'name' | 'logoUrl' | 'experimental_imageUrl'>>;
+  Partial<Pick<OrganizationResource, 'name' | 'logoUrl' | 'imageUrl'>>;
```

Comment:
☁️ if this is an internal Component i think we can omit `logoUrl` from props, else i think we should leave the `logoUrl` but mark it as deprecated somehow.

Reply:
correct, removed as well